docs/reference/configs.rst

```diff
@@ -500,6 +500,8 @@ something like this, where special treatment is given to File pages specifically
 
 .. autodata:: exhale.configs.includeTemplateParamOrderList
 
+.. autodata:: exhale.configs.fullyQualifiedTitles
```

Comment:
not for this file, but in `docs/changelog.rst` could you please add an entry?

```diff
diff --git a/docs/changelog.rst b/docs/changelog.rst
index 71719be..29c5c1a 100644
--- a/docs/changelog.rst
+++ b/docs/changelog.rst
@@ -5,6 +5,12 @@ Changelog
    :local:
    :backlinks: none
 
+v0.3.7
+----------------------------------------------------------------------------------------
+- Add :data:`~exhale.configs.fullyQualifiedTitles` option so that pages like
+  ``device1::error`` and ``device2::error`` render as unique names (:issue:`187`,
+  :pr:`188`).
+
 v0.3.6
 ----------------------------------------------------------------------------------------
 
```

or something similar, just link to new config entry and the issue / pr :slightly_smiling_face:

Reply:
Sure, done.